### PR TITLE
Tune some parameters for RedHat system

### PR DESCRIPTION
**Pull request by strangeman**
Hello. Some default parameters in RHEL and CentOS is more strict than current baseline checks, so I think we should make separate checks for them. I added links for Red Hat Enterprise Linux 6 Security Technical Implementation Guide, looks like its actual for EL 7 too.

### CODE REVIEW

File: controls/os_spec.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@
 # author: Patrick Muench
 
 login_defs_umask = attribute('login_defs_umask', default: '027', description: 'Default umask to set in login.defs')
+login_defs_umask = attribute('login_defs_umask', default: '077', description: 'Default umask to set in login.defs') if os.redhat?
```
Comment (artem-sidorenko) [inline controls/os_spec.rb]: what about `default: os.redhat? ? '077' : '027'` instead of second line?

**Comment by strangeman** [comment]
Done. Should I resolve conflict in my branch, or you can do it directly in PR?

**Comment by artem-sidorenko** [comment]
@strangeman thank you! yes, please [rebase on the latest master and repush](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) the branch

**Comment by strangeman** [comment]
@artem-sidorenko all done, sorry for the delay.

**Pull request merged**